avery_website/templates/layout.html

```diff
@@ -19,13 +19,16 @@
             </a>
         </div>
         <div class="navbar">
-            <div class="container container-center">
+            <div class="container container-top container-center">
                 <a href="{{ url_for('index') }}">Home</a> |
                 <a href="https://www.facebook.com/pg/averyglory/photos">Facebook</a> |
                 <a href="https://www.instagram.com/avery.glory/">Instagram</a> |
                 <a href="{{ url_for('events') }}">Events</a> |
                 <a href="{{ url_for('constitution') }}">Constitution</a> |
-                <a href="{{ url_for('rotation_video') }}">Rotation Video</a>
+            </div>
+            <div class="container container-bottom container-center">
+                <a href="{{ url_for('rotation_video') }}">Rotation Video</a> |
+                <a href="{{ url_for('valentines_video') }}">Valentines Video</a>
```

Comment:
Sorry for initial confusion, this shouldn't be shown on the home page.

Reply:
Fixed [4e368c1](https://github.com/averyhouse/avery-website/pull/3/commits/4e368c13f5a0e9ca80b8320a2727d71eb7585561)